learning/features/assetlib/what_is_assetlib.rst

```diff
@@ -0,0 +1,20 @@
+.. _doc_what_is_assetlib:
+
+About the Asset Library
+=======================
+
+The Godot Asset Library, otherwise known as the AssetLib, is a repository of
+user-submitted Godot addons, scripts and other resources, collectively referred
```

Comment:
Break down your sentences into shorter ones and try to favor direct tense. Example:
`The asset library is a collection of free and open source scripts, add-ons and other assets shared by the community. They're available to all Godot users from within the editor.`